x-pack/plugins/alerting/server/routes/rule/apis/aggregate/types/v1.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export interface DefaultRuleAggregationResult {
```

Comment:
Upon looking at this further, I don't think we need to version this type. Since we are already versioning the `AggregateRulesResponse` type according to: https://github.com/elastic/kibana/pull/164284#discussion_r1323648080.

This types gets formatted then transformed to that versioned this, so I don't think this needs to versioned